fdbserver/storageserver.actor.cpp

```diff
@@ -690,9 +704,24 @@ struct StorageServer {
 		CounterCollection cc;
 		Counter allQueries, getKeyQueries, getValueQueries, getRangeQueries, finishedQueries, lowPriorityQueries,
 		    rowsQueried, bytesQueried, watchQueries, emptyQueries;
-		Counter bytesInput, bytesDurable, bytesFetched,
-		    mutationBytes; // Like bytesInput but without MVCC accounting
+
+		// Bytes of the mutations that have been added to the memory of the storage server. When the data is durable
+		// and cleared from the memory, we do not exact it but add it to bytesDurable.
+		Counter bytesInput;
+		// Bytes of the mutations that have been removed from memory because they durable. The counting is same as
```

Comment:
This is a question mostly for my own edification. From these comments, it seems like `bytesDurable` and `bytesFetched` should be comparable?

Reply:
No. Roughly, `bytesDurable` counted the size of memory that are freed because of the data that are persisted, which is roughly twice as large as the data that are persisted. And more accurately, all size metrics here are slightly different because of counting the paddings of different data structures. `mutationBytes` is more comparable with `bytesFetched` but still not perfect.

That's why I'm adding `kvFetched`, which can be easily comparable with `mutations` (or `setMutations + atomicMutations`), because they all counts number of key-value pairs that changed.

Reply:
Thanks. I think having kv counts is valuable, but we definitely want bytes to be as comparable as possible. In the future, it might be worth considering how we could make the bytes more comparable, but this is clearly a very big step forward.